contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2302,6 +2302,11 @@ antlr_parser_cpp(const char *sourceText)
 			 */
 			auto ssm = std::make_unique<tsqlSelectStatementMutator>();
 			handleBatchLevelStatement(tsql_file->batch_level_statement(), ssm.get());
+
```

Comment:
You may need to make a change in `tsqlUnsupportedFeatureHandler.cpp` file as well.